### DISCUSSION TITLE
Site Name dont change name of site + 404 error

### DISCUSSION THREAD

**Discussion by masterwishx** [alexjustesen/speedtest-tracker#292] [Q&A]
When changing site name  and saving, the name of site is still 'Speedtest Tracker'

also having 404 error: 

![image](https://user-images.githubusercontent.com/28630321/210557691-737f65cf-532f-4aa0-99f9-22ffd9f59cf7.png)

**Comment by masterwishx** [comment]
![image](https://user-images.githubusercontent.com/28630321/210734746-5283dc52-6cc9-4bcc-a196-b9c4fc1311cf.png)

But the Name still Speedtest Tracker

**Reply by alexjustesen** [reply]
To change the site name you'll want to change `APP_NAME` in `.env`. This setting will likely be removed from the UI

**Reply by masterwishx** [reply]
Oh OK, I'm using Unraid docker so I should add variable APP_NAME...

**Reply by alexjustesen** [reply]
PR incoming to keep this in the UI and fix the site name https://github.com/alexjustesen/speedtest-tracker/pull/297
